index.js

```diff
@@ -100,6 +99,8 @@ const createFocusTrap = function (elements, userOptions) {
   //  should be safe code to execute if the `document` option isn't specified
   const doc = userOptions?.document || document;
 
+  const trapStack = userOptions?.trapStack || rooTrapStack;
```

Comment:
I think this can just be
```suggestion
  const trapStack = userOptions?.trapStack || [];
```